LICENSE

```diff
@@ -199,4 +199,44 @@
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
\ No newline at end of file
+   limitations under the License.
+
+================================================================
+
+This product includes a number of Dependencies with separate copyright notices
+and license terms. Your use of these submodules is subject to the terms and
+conditions of the following licenses.
+
+================================================================
+
+================================================================
+Apache-2.0 licenses
+================================================================
+The following components are provided under the Apache-2.0 License.See project link for details.
+The text of each license is the standard Apache 2.0 license.
+
+* prometheus-common(https://github.com/prometheus/common)
+* greptimedb(https://github.com/GreptimeTeam/greptimedb)
+* agatedb(https://github.com/tikv/agatedb)
+* tikv(https://github.com/tikv/tikv)
+* influxdb(https://github.com/influxdata/influxdb)
+* databend(https://github.com/datafuselabs/databend)
+
+File horaemeta/server/service/http/route.go is modified from Prometheus-common.
+Files src/server/src/federated.rs, src/server/src/session.rs are modified from greptimedb.
+Files src/components/skiplist/benches/bench.rs, src/components/skiplist/src/lib.rs, src/components/skiplist/src/key.rs,
+src/components/skiplist/src/list.rs, src/components/skiplist/tests/tests.rs, src/components/skiplist/Cargo.toml are modified from agatedb.
+File src/components/size_ext/src/lib.rs is modified from tikv.
+File src/components/future_ext/src/cancel.rs is modified from influxdb.
+Files src/components/tracing_util/src/lib.rs, src/components/tracing_util/src/logging.rs, src/components/tracing_util/Cargo.toml are modified from databend.
+
+================================================================
+MIT licenses
+================================================================
+The following components are provided under the MIT License. See project link for details.
+The text of each license is also included in licenses/LICENSE-[project].txt
+
+* consistent(https://github.com/buraksezer/consistent)
```

Comment:
You should include the copyright header from https://github.com/buraksezer/consistent/blob/master/LICENSE

Reply:
It's ready here: https://github.com/apache/horaedb/blob/main/licenses/LICENSE-consistent.txt

Should I copy that to this file?

Reply:
that's a reasonable point - it's fine as is, then

@jiacai2050 can you do a new RC? The LICENSE files matter and it would be good for the new release to contain this fix.

Reply:
Yes, I will start a new VOTE threads after cherry pick this patch.